EIPS/eip-1102.md

```diff
@@ -108,7 +97,7 @@ An [open issue](https://github.com/MetaMask/metamask-extension/issues/714) again
 
 ## Backwards compatibility
 
-This proposal impacts dapp authors and requires that they request access to the web3 API before using it. This proposal also impacts developers of web3-enabled environments or dapp browsers as these tools should no longer auto-expose the web3 API; instead, they should only do so if a website requests the API and if the user consents to its access. Environments may continue to auto-expose the web3 API as long as users have the ability to disable this behavior.
+This proposal impacts dapp authors and requires that they request access to a web3 provider before using it. This proposal also impacts developers of web3-enabled environments or dapp browsers as these tools should no longer auto-expose the web3 API; instead, they should only do so if a website requests the provider and if the user consents to its access. Environments may continue to auto-expose the web3 API as long as users have the ability to disable this behavior, but it is not suggested.
```

Comment:
"web3 provider" -> "Ethereum provider API"
"web3-enabled" -> "Ethereum-enabled"
"should only do so if a website requests the provider" -> "should only expose the provider if a website requests the API"

Also, can we remove the "but it is not suggested" text? That's already implied by the "MAY" verbiage in the constraints section.